internal/services/bot/bot_channel_directline_resource_test.go

```diff
@@ -28,6 +28,8 @@ func TestAccBotChannelDirectline_basic(t *testing.T) {
 			Config: r.basicConfig(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("extension_key1").Exists(),
+				check.That(data.ResourceName).Key("extension_key2").Exists(),
```

Comment:
Since this value is `Computed` it will always exist, even if it's empty.
```suggestion
```

Reply:
Done